frontend/src/core/user/components/UserMenu.js

```diff
@@ -173,6 +173,32 @@ export class UserMenuBase extends React.Component<Props, State> {
                     </Localized>
                 </li>
 
+                <li>
+                    <Localized
+                        id="user-UserMenu--github"
+                        glyph={
+                            <i className="fab fa-github fa-fw"></i>
+                        }
+                    >
+                        <a href="https://github.com/mozilla/pontoon/">
+                            { '<glyph></glyph>Hack it on GitHub' }
+                        </a>
+                    </Localized>
+                </li>
+
+                <li>
+                    <Localized
+                        id="user-UserMenu--feedback"
+                        glyph={
+                            <i className="fa fa-comment-dots fa-fw"></i>
+                        }
+                    >
+                        <a href="https://discourse.mozilla.org/c/pontoon">
```

Comment:
Nice work!

You'll also need to add `rel="noopener noreferrer" target="_blank"` to the `<a>` elements, e.g.:
```html
<a
    href="https://github.com/mozilla/pontoon/"
    rel="noopener noreferrer"
    target="_blank"
>
    { '<glyph></glyph>Hack it on GitHub' }
</a>
```

Please do that for Terms, GitHub, Feedback and Help links.

And while you're at it, please remove the Top Contributors and the Machinery links (as well as their translations from the .ftl files).

Reply:
Done. @mathjazz